test/plugins/features/movies/controller.test.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+
+const Knex = require('../../../../lib/libraries/knex.js');
+
+const Controller = require('../../../../lib/plugins/features/movies/controller');
+const Movie = require('../../../../lib/models/movie.js');
+
+describe('movie controller', () => {
+
+  beforeEach('clear tables before test start', async () => {
+
+    await Knex.raw('TRUNCATE movies CASCADE');
+
+  });
+
+  describe('create', () => {
```

Comment:
Rather than `create` here, I'd use `POST` as you're describing the POST endpoint

Reply:
Isn't this at the controller level though? I'm using `POST` at the `index` level.

Reply:
Ahh, yeah, I'm sorry, that makes sense